src/assets/styles/variables.less

```diff
@@ -18,4 +18,4 @@
 @expanded-width: 300px;
 
 // Heights
-@splitter-height: 2px;
\ No newline at end of file
+@splitter-height: 1px;
```

Comment:
The design did specify a 2px thick splitter (at least, that's what Rui said before)